apps/mobile/mobile-app/app/+html.tsx

```
@@ -1,39 +1,39 @@
-            __html: responsiveBackground,
-          }}
-        />
-        {/* Add any additional <head> elements that you want globally available on web... */}
-      </head>
-      <body>{children}</body>
-    </html>
-  );
+				<ScrollViewStyleReset />
+				{/* Using raw CSS styles as an escape-hatch to ensure the background color never flickers in dark-mode. */}
+				<style
+					dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)